src/main/java/com/epam/controller/Controller.java

```diff
@@ -0,0 +1,36 @@
+package com.epam.controller;
+
+
+import com.epam.model.data.BaseModel;
+
+public class Controller {
+    private BaseModel model;
+
+    public Controller(BaseModel model) {
+        this.model = model;
+    }
+
+    public int getParagraphCountInfo() {
```

Comment:
get - use only for getters in public methods, try to rename methods using more meaningful verb (e.g. `countParagraphs`)